packages/core/lib/debug/printer.js

```diff
@@ -86,6 +88,16 @@ class DebugPrinter {
     );
   }
 
+  warnIfNoSteps() {
+    if (this.session.view(trace.steps).length === 0) {
+      this.config.logger.log(
+        colors.bold(
+          "Warning: this transaction has no trace steps. This may happen if you are attempting to debug a transaction sent to an externally-owned accounts, or if the node you are connecting to failed to produce a trace for some reason. Please check your configuration and try again."
```

Comment:
```suggestion
          "Warning: this transaction has no trace steps. This may happen if you are attempting to debug a transaction sent between externally-owned accounts, or if the node you are connecting to failed to produce a trace for some reason. Please check your configuration and try again."
```

Reply:
Yeah, I changed that to `to an` because that's what's really relevant, is who the transaction is sent to.  Transactions are *always* sent from an externally-owned account; it seems a little silly to specify that part.